release_notes/addtl-release-notes.adoc

```diff
@@ -0,0 +1,96 @@
+
+|L::
+xref:../observability/logging/logging_release_notes/logging-5-9-release-notes.adoc#logging-5-9-release-notes[{logging-uc}]
+
+|M::
+xref:../migration_toolkit_for_containers/release_notes/mtc-release-notes.adoc#mtc-release-notes[{mtc-full} ({mtc-short})]
+
+|N::
+xref:../observability/network_observability/network-observability-operator-release-notes.adoc#network-observability-operator-release-notes[Network Observability Operator] +
+{nbsp} +
+xref:../security/nbde_tang_server_operator/nbde-tang-server-operator-release-notes.adoc#nbde-tang-server-operator-release-notes[Network-bound Disk Encryption (NBDE) Tang Server Operator]
+
+|O::
+xref:../backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.adoc#oadp-1-4-release-notes[{oadp-first}] +
+{nbsp} +
+link:https://docs.redhat.com/en/documentation/red_hat_openshift_dev_spaces/3.16#Release%20Notes[{openshift-dev-spaces-productname}] +
+{nbsp} +
+xref:../observability/distr_tracing/distr-tracing-rn.adoc#distr-tracing-rn[{DTProductName}] +
+{nbsp} +
+link:https://docs.redhat.com/en/documentation/red_hat_openshift_gitops/1.13/html/release_notes/index[{gitops-title}] +
+{nbsp} +
+link:https://docs.redhat.com/en/documentation/red_hat_openshift_local/[{openshift-local-productname}] +
+{nbsp} +
+link:https://docs.redhat.com/en/documentation/openshift_sandboxed_containers/1.6/html/release_notes/index[{osc}] +
+{nbsp} +
+link:https://docs.redhat.com/en/documentation/red_hat_openshift_serverless/1.33/html/about_openshift_serverless/serverless-release-notes#serverless-release-notes[Red Hat {ServerlessProductName}] +
+{nbsp} +
+link:https://docs.redhat.com/en/documentation/red_hat_openshift_pipelines/1.15/html/about_openshift_pipelines/op-release-notes[{pipelines-title}] +
+{nbsp} +
+xref:../virt/release_notes/virt-release-notes-placeholder.adoc#virt-release-notes-placeholder[Red Hat {VirtProductName}] +
+{nbsp} +
+xref:../observability/otel/otel-rn.adoc#otel-rn[{OTELName}]
+
+|P::
+xref:../observability/power_monitoring/power-monitoring-release-notes.adoc#power-monitoring-release-notes[{PM-title-c}]
+
+|R::
+xref:../nodes/pods/run_once_duration_override/run-once-duration-override-release-notes.adoc#run-once-duration-override-release-notes[{run-once-operator}]
+
+|S::
+xref:../nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc#nodes-secondary-scheduler-release-notes[{secondary-scheduler-operator-full}] +
+{nbsp} +
+xref:../security/security_profiles_operator/spo-release-notes.adoc#spo-release-notes[Security Profiles Operator] +
+{nbsp} +
+xref:../service_mesh/v2x/servicemesh-release-notes.adoc#service-mesh-release-notes[{SMProductName} 2.x] +
+{nbsp} +
+link:https://docs.openshift.com/service-mesh/3.0.0tp1/ossm-release-notes/ossm-release-notes-assembly.html[{SMProductName} 3.x]
+
+|W::
+xref:../windows_containers/wmco_rn/windows-containers-release-notes-10-17-x.adoc#windows-containers-release-notes-10-17-x[{productwinc}]
```

Comment:
Same here, shouldn't this be under "O" for "OpenShift" not sure if you'd then alphabetize it next by "s" in "support" or w in "Windows"?

This is why I think the portal alphabetizing is confusing :sweat_smile: